### PR TITLE
Fix talentos' toggle button

### PR DESCRIPTION
**Pull request by MarcosNASA**
Co-authored-by: Addin <juanprmendoza96@gmail.com>

### CODE REVIEW

**Comment by Addin** [comment]
Fixed conflicts, waiting for review and approval to merge. 👀

**Review by Santiago-j-s — APPROVED** [review]

**Review by Santiago-j-s — APPROVED** [review]
LGTM

**Pull request merged**
